server/start.go

```diff
@@ -21,17 +32,6 @@ import (
 	tmtypes "github.com/tendermint/tendermint/types"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
-
-	"github.com/cosmos/cosmos-sdk/client"
```

Comment:
Why the move of these imports? They were intentionally down in this block using the popular Go convention for blocks of:
1. Go standard library imports
2. External dependencies
3. Same repository dependencies

Please revert

Reply:
IDK how others feel, but my general take on import orders is that it should have it be enforced via linter / or on-save tooling. Manual import ordering is a lot of extra mental energy for reviewer + pr writer

Reply:
Yeah I'm used to the standard of
1, 3, 2

I will change this back. Would be great to have it enforced with linter because it's inconsistent throughout the codebase

Reply:
golint actually emphasizes the order I suggested, though I personally hate golint for being pedantic, but anyways in most code bases it is the order I kindly suggested.

Reply:
Also one other thing, if we are changing import styles in a massive PR, that just creates unnecessary noise to review. Thanks for the reviert @p0mvn!